src/jaxsim/mujoco/loaders.py

```diff
@@ -1,7 +1,7 @@
 import pathlib
 import tempfile
 import warnings
-from typing import Any
+from typing import Any, Callable
```

Comment:
Do you think it's worth creating a new type so that it's more clear that this function is something like $\mathcal{H}: (x, y) \mapsto z$ ?

```py
HeightmapCallable = Callable[[jtp.FloatLike, jtp.FloatLike], jtp.FloatLike]
```

Reply:
Sure, it looks good!